src/index.js

```diff
@@ -68,25 +72,25 @@ const RemoveDataTestIds = ({ types: t }) => {
 
       const isObjectDefined = value => Object.keys(value).length !== 0;
 
-      const matchAndReplaceValues = currentAttr => {
-        const {
-          value: { expression: { properties = [] } = {} } = {}
-        } = currentAttr;
+      const matchAndReplaceValues = ({ value = {}, ...currentAttr }) => {
+        const { expression: { properties = [] } = {} } = value || {};
```

Comment:
isn't value defaulted on line 75 `value = {}` so you shouldn't need `value || {}` on line 76 ??

Reply:
Not necessarily because the default assignment only happens for values that are undefined. It does not run the default assignment if the value is null which is why I do that OR operation

Reply:
ah right, gotta love javascript